app/Resources/views/macros/forms.html.twig

```diff
@@ -81,7 +81,7 @@
             <span class="text-muted">({{ msg("optional") }})</span>
         {% endif %}
     </label>
-    <input type="date" class="form-control" name="{{ type }}" placeholder="2016-11-01"{% if required %} required{% endif %} />
+    <input type="date" class="form-control" name="{{ type }}" placeholder="YYYY-MM-DD"{% if required %} required{% endif %} />
```

Comment:
Actual date that will be submitted? You mean if I pass in the start/end date via URL, it should populate it in the form? We can make that happen